commit/core.pyx

```diff
@@ -207,6 +210,64 @@ cdef class Evaluation :
         LOG( '   [ %.1f seconds ]' % ( time.time() - tic ) )
 
 
+    def load_confidence_map( self, confidence_map_filename ) :
```

Comment:
This function is useless and not in line with the conventions used so far. Till now, all functions in the `core.pyx` file are intended as functions to expose functionality that users directly call in the form `mit.FUNCTION()`. This function is instead called inside `mit.fit()`, and is called only once, so it is not necessary. Move all the code from this function to the right place (i.e. either `fit` or `solvers.solve()`)

Reply:
I moved the code of the function inside  `mit.fit()`. 

I did not want to add loadings and validations inside `mit.fit()`.  Maybe we can keep this function to load the data and add a flag in `mit.fit()` to use it or not.